packages/core/src/awsService/ec2/sshKeyPair.ts

```diff
@@ -35,13 +38,25 @@ export class SshKeyPair {
     }
 
     public static async generateSshKeyPair(keyPath: string): Promise<void> {
```

Comment:
Do we have a sanity check somewhere that ensures we don't overwrite a path in the user's ~/.ssh ? I assume we are writing to the extension's own globalStorage, but maybe wherever we do the "delete timer" we should double-check.

Reply:
So far `SshKeyPair` is only used with globalStorage, but nothing is stopping it from being used elsewhere. To avoid messing with user's `~/.ssh` (or other sensitive) we could have a list of blacklisted paths, and throw an error if the `SshKeyPair` is called with that any of those paths as a argument. Alternatively, we could force the path to be a subpath of globalStorage. I can track that and make it a follow up.